src/coreclr/vm/eeconfig.cpp

```diff
@@ -480,7 +480,10 @@ HRESULT EEConfig::sync()
     }
 
     pReadyToRunExcludeList = NULL;
+
 #if defined(FEATURE_READYTORUN)
+    fReadyToRun = CLRConfig::GetConfigValue(CLRConfig::EXTERNAL_ReadyToRun);
```

Comment:
Could use `fReadyToRun` below instead of going through `g_pConfig` to make the dependency clear

Reply:
Sounds good